tests/test_WavelengthFitData.py

```diff
@@ -18,37 +18,41 @@ class WavelengthFitDataTestCase(lsst.utils.tests.TestCase):
     def setUp(self):
         self.length = 128
         self.fiberId = np.arange(123, 456, 78, dtype=np.int32)
-        self.pixels = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.nominalXPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitXPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.nominalPixelPos = np.random.uniform(size=self.fiberId.shape)*self.length
+        self.fitPixelPosErr = np.random.uniform(size=self.fiberId.shape)*self.length*0.01
         self.wlMin = 600.0  # nm
         self.wlMax = 900.0  # nm
         scale = self.wlMax - self.wlMin
-        self.measuredWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
+        self.fitWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
         self.actualWavelength = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
-        self.lines = [LineData(*args) for args in zip(self.fiberId, self.pixels, self.measuredWavelength,
-                                                      self.actualWavelength)]
+        self.fitWavelengthErr = (np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin)*0.0
+        self.wavelengthCorr = (np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin)*0.01
+        self.reflines = np.random.uniform(size=self.fiberId.shape)*scale + self.wlMin
+        self.status = np.random.uniform(size=self.fiberId.shape)
+        self.lines = [LineData(*args) for args in zip(self.fiberId, self.reflines,self.nominalXPixelPos, self.fitXPixelPos, self.nominalPixelPos, self.fitPixelPos, self.fitWavelength, self.fitPixelPosErr, self.fitWavelengthErr,self.wavelengthCorr, self.status)]
 
     def assertWavelengthFitData(self, wlFitData, atol=0.0):
         """Check that the WavelengthFitData is what we expect
-
         Parameters
         ----------
         wlFitData : `pfs.drp.stella.calibrateWavelengthsTask.WavelengthFitData`
             Object to check.
         atol : `float`
-            Absolute tolerance for ``measuredWavelength`` (might not be exact
-            if calculated).
+            Absolute tolerance for ``fitWavelength`` (might not be exact
         """
         self.assertEqual(len(wlFitData), len(self.fiberId))
         self.assertFloatsEqual(wlFitData.fiberId, self.fiberId)
-        self.assertFloatsEqual(wlFitData.pixels, self.pixels)
-        self.assertFloatsAlmostEqual(wlFitData.measuredWavelength, self.measuredWavelength, atol=atol)
-        self.assertFloatsEqual(wlFitData.actualWavelength, self.actualWavelength)
-
+        self.assertFloatsAlmostEqual(wlFitData.fitWavelength, self.fitWavelength, atol=atol)
```

Comment:
Why did you remove three tests and only add one? You added more functionality, so you should be adding more tests.

Reply:
I added several lines. 
I'll make more lines if needed.

Reply:
Please add tests for all the elements of `WavelengthFitData`.